### PR TITLE
added pravinshedbale.json

### PR DESCRIPTION
**Pull request by pravinshedbale**
added user 'pravinshedbale' in submissions

### CODE REVIEW

File: Submissions/pravinshedbale.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Pravin Shedbale",
+    "img": "[YOUR_IMG_URL]",
```
Comment (donmarvex) [inline Submissions/pravinshedbale.json:3]: Hello @pravinshedbale , please add your image url. Go through the steps listed in order for you to be able to add your image url.

**Review by donmarvex — CHANGES_REQUESTED** [review]
Hello @pravinshedbale , you'll need to make the changes as requested below

**Review by pravinshedbale — COMMENTED** [review]
Image URL added

**Comment by donmarvex** [comment]
> @donmarvex Image URL added

Thank You @pravinshedbale , you're good to go!

**Review by donmarvex — APPROVED** [review]

**Comment by pravinshedbale** [comment]
Thanks @donmarvex !!

**Pull request merged**